u2f-gae-demo/war/js/u2fdemo.js

```diff
@@ -75,22 +80,10 @@ function hideMessage() {
 
 function highlightTokenCardOnPage(token) {
   console.log(token);
-
-  var cardChildren = document.getElementById(token.public_key).children;
-  for (i = 0; i < cardChildren.length; i++) {
-    if ($(cardChildren[i]).hasClass("cardContent")) {
-      $(cardChildren[i]).addClass("highlight");
-    }
-  }
-
-  window.setTimeout(
-    function() {
-      for (i = 0; i < cardChildren.length; i++) {
-        $(cardChildren[i]).removeClass("highlight", 2000);
-      }
-    },
-    500
-  );
+  var cardContent = $("#" + token.public_key).find(".cardContent");
```

Comment:
Can we revert this change?  It's awkward, but needed because iOS can't handle the jQuery for some reason.